wallet/src/Pos/Util/BackupPhrase.hs

```diff
@@ -29,17 +29,36 @@ newtype BackupPhrase = BackupPhrase
     { bpToList :: [Text]
     } deriving (Eq, Generic)
 
+data MnemonicWordCount
+    = Nine
+    | Twelve
+    | Fifteen
+    | Eighteen
+    | Twentyone
+    | Twentyfour
+    deriving (Eq, Show)
+
+byteCount :: MnemonicWordCount -> Int
```

Comment:
It would be nice to post here the link to the BIP39 proposal, so that people know where we picked this magic numbers from 😉 

We should really call this function _checksumLength_ in my opinion, as this is the size of the checksum, in reality.